CHANGES.md

```diff
@@ -1,5 +1,31 @@
 # RxJava Releases #
 
+### Version 1.0.0 - August 5th 2015 ####
+
+Initial stable release!
+
+In order to provide a library that no project using RxJava would hesitate to depend on, the decision
+was made to remove the APIs which were not absolutely fundamental to all apps. That is what's
+contained in this release.
+
+Functionality which was previously part of this library is being explored in separate, modular
+libraries. This allows for a simpler process of design, development, and experimentation for the
+best ways to provide features like hooks into the lifecycle, binding to UI components, and
+simplifying interaction with all of Android's API. Not only can these projects now have their own
+release schedule, but it allows developers to pick and choose which ones are appropriate for your
+application.
+
+Applications using the various APIs which were previously in this library do not need to update
+immediately. Due to the number of APIs removed, switching to 1.0 and the use of these third-party
+libraries should be done gradually.
+
+Breaking changes:
+
+ * `AndroidSchedulers.handlerThread()` is now `HandlerScheduler.from()`.
+ * All other APIs have been removed aside from `AndroidSchedulers.mainThread()`, `RxAndroidPlugins`,
```

Comment:
Maybe we should bold this sentence so that it really sinks in, since it's the one-line summary of everything.